src/sentry/search/events/datasets/field_aliases.py

```diff
@@ -158,3 +158,11 @@ def resolve_replay_alias(builder: BaseQueryBuilder, alias: str) -> SelectType:
         [Function("nullif", [builder.column(column), ""]) for column in columns],
         alias,
     )
+
+
+def resolve_column_if_exists(builder: BaseQueryBuilder, alias: str) -> SelectType:
+    if hasattr(builder, "resolve_tag_key"):
+        hasColumn = builder.resolve_tag_key(alias)
+        if hasColumn:
+            return builder.column(alias)
+    return Function("nullif", ["", ""], alias)
```

Comment:
I might be missing something obvious, but I couldn't find an easy way to just return a null value.

Reply:
That seems to work, thanks for the suggestion! Can't believe it was so obvious 😄